appdata.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>org.robomongo.Robo3T</id>
+    <launchable type="desktop-id">org.robomongo.Robo3T.desktop</launchable>
+    <name>Robo 3T</name>
+
+    <metadata_license>GPL-3.0</metadata_license>
+    <project_license>LicenseRef=https://raw.githubusercontent.com/Studio3T/robomongo/master/LICENSE</project_license>
```

Comment:
this isn't a valid SPDX code. you want `GPL-3.0-only` https://spdx.org/licenses/